frontend/src/pages/categories/questions/[id].tsx

```diff
@@ -100,28 +100,29 @@ const CategoryQuestions: NextPage = (): JSX.Element => {
 
   return (
     <Layout metaTitle={categoryQuestions?.title}>
-      {!categoryQuestions ? (
-          <div className="flex justify-center w-full py-8">
-            <Spinner className="w-6 h-6 text-red-500" />
-          </div>
-        ) : (
+      {!categoryQuestions 
+      ? <Loading />
+      : (
           <>
-            <div className="pt-5 pb-4 bg-white rounded-md shadow-primary">
-              <h1 className="text-2xl font-bold text-red-500 text-center">"{categoryQuestions?.title}"</h1>
+            <div css={styles.category_title}>
+              <h1>"{categoryQuestions?.title}"</h1>
             </div>
-            <form onSubmit={handleSubmit(handleSave)} className="mt-4 bg-white px-8 py-4 rounded-md shadow-primary">
-              <div className="divide-y divide-gray-100">
+            <form 
+              onSubmit={handleSubmit(handleSave)} 
+              css={styles.question_form}
+            >
+              <div>
                 {categoryQuestions?.questions.map((question: ICategoryQuestion, questionIdx: number) => (
-                  <div key={questionIdx} className="flex flex-col pb-10 pt-4">
-                    <div className="flex items-start flex-col md:flex-row">
-                      <div className="md:w-1/2 flex flex-col">
-                        <label className="text-lg font-semibold">{questionIdx+1}. {question?.value}</label>
+                  <div key={questionIdx} css={styles.question_wrapper}>
+                    <div>
+                      <div>
+                        <label>{questionIdx+1}. {question?.value}</label>
                       </div>
-                      <div className="mt-3 md:mt-0 md:w-1/2 space-y-4">
+                      <div css={styles.choices_container}>
                         {question?.choices?.map((choice: IChoice, idx: number) => (
-                          <div key={idx} className="flex space-x-2 items-center">
-                            <span className="font-medium">{convertIndexToAlphabet(idx+1) + ". "}</span>
-                            <div className="flex items-center">
+                          <div key={idx} css={styles.choices_wrapper}>
+                            <span>{convertIndexToAlphabet(idx+1) + ". "}</span>
```

Comment:
So that it will convert the looped index into alphabet of the choices given
![image](https://user-images.githubusercontent.com/108642414/188585001-7e8715ac-5018-4c17-a24e-2c8ad73f9a1a.png)